gaia-header.js

```diff
@@ -404,7 +404,6 @@ module.exports = component.register('gaia-header', {
    */
   onUpdate: function() {
     debug('onUpdate');
-    this.shadowRoot.firstChild.dir = dir();
```

Comment:
@wilsonpage now that `onUpdate` is useless, do you want me to remove it completely (along with the `dirchanged` listeners) or should we leave that here to ease future work?